pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -868,6 +868,11 @@ func createURIMatch(match k8s.HTTPRouteMatch) (*istio.StringMatch, *ConfigError)
 	}
 	switch tp {
 	case k8s.PathMatchPrefix, k8s.PathMatchImplementationSpecific:
+		if dest == "/" {
+			return &istio.StringMatch{
+				MatchType: &istio.StringMatch_Regex{Regex: regexp.QuoteMeta(dest)},
```

Comment:
should this be `&istio.StringMatch_Prefix`?

Reply:
Thank you. Updated.